python/README.md

```diff
@@ -0,0 +1,54 @@
+Python connection to AWS ElastiCache Redis
+===========================================
+
+An example app connecting to Redis using [redis-py](https://github.com/andymccurdy/redis-py)
+
+## Migrating from your Redis service
+
+The follow code snippet shows the differences between connecting to the old Redis service
+to the new AWS ElastiCache Redis service. The following app code shows how to configure
+the Redis client with the Redis service bound to the app. See
+[redis-py docs](https://github.com/andymccurdy/redis-py#ssl-connections) for more information.
+
+```python
+import os
+import json
+import redis
+
+
+##
+## The old Redis service `redis32`
+##
+redis_config = dict()
+services = json.loads(os.getenv('VCAP_SERVICES'))
+redis_credentials = services['redis32'][0]['credentials']
+
+redis_config['host'] = redis_credentials['hostname'] ## Reassign hostname to host
+redis_config['port'] = redis_credentials['port']
+redis_config['password'] = redis_credentials['password']
+
+client = redis.Redis(**redis_config)
```

Comment:
Should we move this line to go below the new example config, to make it clear that the client needs to be instantiated in either scenario?